### PR TITLE
My feature

### PR DESCRIPTION
**Pull request by astro-satyam75**
### Description

Added `.boxplot()` to the matplotlib folder inside the content folder. 

In the boxplot.md file, added the definition, description, syntax and example use cases for the boxplot() method in pandas.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/matplotlib/concepts/pyplot/terms/boxplot/boxplot.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
+Title: '.subplots()'
```
Comment (caupolicandiaz) [inline content/matplotlib/concepts/pyplot/terms/boxplot/boxplot.md:2]:
```suggestion
Title: '.boxplot()'
```

File: content/matplotlib/concepts/pyplot/terms/boxplot/boxplot.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+Description: 'Draws a box and whisker plot.'
```
Comment (caupolicandiaz) [inline content/matplotlib/concepts/pyplot/terms/boxplot/boxplot.md:3]:
```suggestion
Description: 'Returns a box and whisker plot.'
```

File: content/matplotlib/concepts/pyplot/terms/boxplot/boxplot.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+Tags:
+  - 'Graphs'
+  - 'Libraries'
+  - 'Matplotlib'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`.boxplot()`** is a method in the matplotlib library returns a box and whisker plot by taking one or more arrays of data as input.
```
Comment (caupolicandiaz) [inline content/matplotlib/concepts/pyplot/terms/boxplot/boxplot.md:16]:
```suggestion
The **`.boxplot()`** is a method in the Matplotlib library that returns a box and whisker plot based on one or more arrays of data as input.
```

File: content/matplotlib/concepts/pyplot/terms/boxplot/boxplot.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+## Syntax
+
+```pseudo
+matplotlib.pyplot.boxplot(x, notch, sym, vert, whis, bootstrap, usermedians, conf_intervals, positions, widths, patch_artist, labels, manage_ticks, autorange, meanline, zorder )
+```
+
+The 'x' parameter is required, and represents an array or a sequence of vectors. Other parameters are optional and used to modify the features of the boxplot.
```
Comment (caupolicandiaz) [inline content/matplotlib/concepts/pyplot/terms/boxplot/boxplot.md:24]:
```suggestion
The `x` parameter is required, and represents an array or a sequence of vectors. Other parameters are optional and used to modify the features of the boxplot.
```

File: content/matplotlib/concepts/pyplot/terms/boxplot/boxplot.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+`.boxplot()` takes the following arguments:
+
+- `x` : Takes in the data to be plotted as a list, array, or a sequence of arrays. Each array represents a different dataset to be plotted in the boxplot.
+- `notch`:  If `True`, a notch is drawn around the median to give an idea of the median uncertainty.
```
Comment (caupolicandiaz) [inline content/matplotlib/concepts/pyplot/terms/boxplot/boxplot.md:29]:
```suggestion
- `notch`:  If `True`, a notch is drawn around the median to represent the median uncertainty.
```

File: content/matplotlib/concepts/pyplot/terms/boxplot/boxplot.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+- `sym`: This parameter is used to specify the symbol to be used to represent outliers. By default, outliers are represented as dots.
```
Comment (caupolicandiaz) [inline content/matplotlib/concepts/pyplot/terms/boxplot/boxplot.md:30]:
```suggestion
- `sym`: A parameter used to modify the designation of outliers. By default outliers are represented as dots, if an empty string is passed any outliers in the data will not be visible in the plot.
```

File: content/matplotlib/concepts/pyplot/terms/boxplot/boxplot.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+# Create a box and whisker plot
+plt.boxplot(data)
+
+# Show the plot
+plt.show()
+```
+
+Output:
+
+![Output of matplotlib.pyplot.boxplot() method example 1](https://raw.githubusercontent.com/astro-satyam75/docs/my_feature/media/matplotlib-boxplot-example1-output.png)
```
Comment (caupolicandiaz) [inline content/matplotlib/concepts/pyplot/terms/boxplot/boxplot.md:63]: The image below doesn't match this example. Please update with the correct file. Also, ideally the plot should be exported at a minimum resolution of 300dpi:
plt.savefig('boxplot_fig', dpi=300)

**Review by caupolicandiaz — APPROVED** [review]
@astro-satyam75 I've gone ahead and made the suggested changes in order to keep this entry moving along. Now, this is ready for a second review.

File: content/matplotlib/concepts/pyplot/terms/boxplot/boxplot.md [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+# Create a box and whisker plot
+plt.boxplot(data)
+
+# Show the plot
+plt.show()
+```
+
+Output:
+
+![Output of matplotlib.pyplot.boxplot() method example 1](https://raw.githubusercontent.com/astro-satyam75/docs/my_feature/media/matplotlib-boxplot-example-1.png)
```
Comment (SSwiniarski) [inline content/matplotlib/concepts/pyplot/terms/boxplot/boxplot.md:63]:
```suggestion
![Output of matplotlib.pyplot.boxplot() method example 1](https://raw.githubusercontent.com/Codecademy/docs/main/media/matplotlib-boxplot-example-1.png)
```

File: content/matplotlib/concepts/pyplot/terms/boxplot/boxplot.md [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+plt.ylabel('Group')
+plt.title('Customized box and whisker plot')
+
+# Show the plot
+plt.show()
+```
+
+Output:
+
+![Output of matplotlib.pyplot.boxplot() method example 2](https://raw.githubusercontent.com/astro-satyam75/docs/my_feature/media/matplotlib-boxplot-example-2.png)
```
Comment (SSwiniarski) [inline content/matplotlib/concepts/pyplot/terms/boxplot/boxplot.md:86]:
```suggestion
![Output of matplotlib.pyplot.boxplot() method example 2](https://raw.githubusercontent.com/Codecademy/docs/main/media/matplotlib-boxplot-example-2.png)
```

**Review by SSwiniarski — APPROVED** [review]

**Comment by SSwiniarski** [comment]
@caupolicandiaz, I changed the links to the images and it looks good to merge now.

**Comment by SSwiniarski** [comment]
@astro-satyam75, congrats on the new entry. Live link can be found here: https://www.codecademy.com/resources/docs/matplotlib/pyplot/boxplot

image

**Pull request merged**